client/settings/payment-method-settings/payment-request-button-preview.js

```diff
@@ -0,0 +1,81 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { React } from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+import { Elements } from '@stripe/react-stripe-js';
+import { loadStripe } from '@stripe/stripe-js';
+
+/**
+ * Internal dependencies
+ */
+import PaymentRequestDemoButton from './payment-request-demo-button';
+import {
+	getPaymentRequestData,
+	shouldUseGooglePayBrand,
+} from 'payment-request/utils';
+
+/**
+ * stripePromise is used to pass into <Elements>'s stripe props.
+ * The stripe prop in <Elements> can't be change once passed in.
+ * Keeping this outside of <PaymentRequestButtonPreview> so that
+ * re-rendering does not change it.
+ */
+const stripeSettings = getPaymentRequestData( 'stripe' );
+const stripePromise = loadStripe( stripeSettings.publishableKey, {
+	stripeAccount: stripeSettings.accountId,
+	locale: stripeSettings.locale,
+} );
```

Comment:
Although this means that this is initialized as soon as the file is included - what do you think about putting this in a `useMemo` callback inside the component's body, so that it doesn't change and it is unloaded when the component is not rendered?

```
const stripePromise = useMemo( () => {
		const stripeSettings = getPaymentRequestData( 'stripe' );
		return loadStripe( stripeSettings.publishableKey, {
			stripeAccount: stripeSettings.accountId,
			locale: stripeSettings.locale,
		} );
	}, [] );
```

Reply:
> so that it doesn't change and it is unloaded when the component is not rendered?

I like this idea. I am also now wondering if this should be inside `useEffect`. Going to take a look.